transaction/std/src/memo/gift_code_cancellation.rs

```diff
@@ -23,17 +26,52 @@ impl RegisteredMemoType for GiftCodeCancellationMemo {
 }
 
 impl GiftCodeCancellationMemo {
-    /// The length of the custom memo data
+    /// Number of bytes in the memo
     pub const MEMO_DATA_LEN: usize = 64;
 
-    /// Create a new gift code memo
-    pub fn new(global_index: u64) -> Self {
-        GiftCodeCancellationMemo::from(global_index)
+    /// Number of bytes used to represent the global index of the TxOut used to
+    /// fund the cancelled gift code
+    pub const INDEX_DATA_LEN: usize = 8;
+
+    /// Number of bytes used to represent the fee paid for gift code
+    /// cancellation
+    pub const FEE_DATA_LEN: usize = 7;
```

Comment:
i think we only needed to save 1 byte, so it was suggested to record 7 bytes for the fee instead of 8.

(https://github.com/mobilecoinfoundation/mcips/pull/4)

Reply:
I originally thought smaller byte amounts might be fine as well, but fees are paid in picomob there are 1^12 picomob in supply, so 5 bytes would only be able to record up to 1 mob. Other token combos of large supplies + extremely small minimum units could quickly fill up 5 or 6 bytes.